src/ResultPage.js

```diff
@@ -0,0 +1,81 @@
+import styled from 'styled-components/macro'
+import Button from './components/Button'
+import { useState, useEffect } from 'react'
+
+function ResultPage({countyData, resetSearch}) {
+
+    const [countyRank, setCountyRank] = useState('')
+
+    useEffect(() => rankCountyIncidence(), [countyData])
+
+    function rankCountyIncidence() {
```

Comment:
This function doesn't actually rank (as in "put in order" / "ranking"), right. It just classifies whether it's green or red, or not? Maybe you'd rather name the function `classifyCountyIncidence`. And then of course `[countyClassificaton, setCountyClassification]`

Reply:
thanks, I agree, that fits better than "rank".